Backend/src/schema/resolvers.js

```diff
@@ -0,0 +1,554 @@
+const db = require('../db/database');
+const utils = require('../utils/utils');
+const { ListItem, ListItemInfo } = require("../dto/listItem");
+const { User, UserInfo } = require('../dto/user');
+const jwtService = require('../jwt/service');
+
+
+const resolvers = {
+    Query: {
+        getOneListItem: async function (parent, args, context, info) {
+            let listItemInfo;
+            let listItem;
+            let user;
+
+            const session = context.driver.session();
+
+            try {
+                let writeTxResultPromise = session.writeTransaction(async txc => {
+                    let result = await txc.run(
+                        `
+                        MATCH (listItem: ListItem { id: $id })
+                        RETURN listItem
+                         `,
+                            { id: Number(args.id) },
+                        )
+                    if (result != null) {
+                        return result.records.map(record => ({
+                            listItem: record.get('listItem').properties,
+                        }))
+                    }
+                    return;
+                })
+                let txResult = await writeTxResultPromise
+
+                listItem = txResult[0] != undefined ? txResult[0].listItem : null;
+                
+                if (listItem != null) {
+                    writeTxResultPromise = session.writeTransaction(async txc => {
+                        result = await txc.run(
+                            `
+                        MATCH (listItem: ListItem { id: $id })
+                        MATCH (user:User)<-[:ASSIGNED]-(listItem)
+                        RETURN user
+                         `,
+                            { id: Number(args.id) },
+                        )
+                        if (result != null) {
+                            return result.records.map(record => ({
+                                user: record.get('user') != undefined ? record.get('user').properties : null
+                            }))
+                        }
+                        return;
+                    })
+                    let txResult = await writeTxResultPromise
+
+                    user = txResult[0] != undefined ? txResult[0].user : null;
+
+                    listItem.assignee = user;
+                    
+                    listItemInfo = new ListItemInfo(listItem);
+                }
+                return listItemInfo;
+            } finally {
+                session.close()
+            }
+        },        
+        getAllListItems: async function(parent, args, context, info) {
+            let allListItems = [];
+            const session = context.driver.session();
+
+            cypher = 'MATCH(allListItems: ListItem) ';
+            if(typeof(args.isDone) === 'boolean') {
+                cypher+= 'WHERE(allListItems.isDone = '+ args.isDone +') ';
```

Comment:
No cypher injection vulnerabilities, please!
```suggestion
                cypher+= 'WHERE( allListItems.isDone = $isDone )`;
```